canonical-web-and-design/ubuntu.com
# Visual improvements to subscription table

## Done

- Fix /advantage 's table alignment according to this [design](https://app.zeplin.io/project/601a8f0a371bc62234c9fc39/screen/601bbaaafb4831787d52354b)

## QA

- go to https://ubuntu-com-9220.demos.haus/advantage?test_backend=true
- login
- check that the table looks as specified


## Issue / Card

Fixes https://github.com/canonical-web-and-design/master-epics/issues/245

## Screenshots

before 
![image](https://user-images.githubusercontent.com/11927929/107397607-1b2beb80-6aff-11eb-95fc-b6e8d0f7cf10.png)

after

![image](https://user-images.githubusercontent.com/11927929/107397821-4a425d00-6aff-11eb-9044-533833ec46a3.png)


Small screen
![image](https://user-images.githubusercontent.com/11927929/107397868-59290f80-6aff-11eb-8b04-7595c1d33901.png)
